apps/app/src/lib/near-social.ts

```
@@ -0,0 +1,72 @@
+      body: JSON.stringify({
+        keys,
+      }),
+    });
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+
+    const data = await response.json();
+
+    if (data && data[accountId] && data[accountId].profile) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)